pkg/controllers/dynakube/logmodule/daemonset/container.go

```diff
@@ -0,0 +1,83 @@
+package daemonset
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/address"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	defaultImageRepo = "registry.lab.dynatrace.org/oneagent/dynatrace-logmodule-amd64" // TODO: finalize
```

Comment:
what do you mean by finalize?

Reply:
that's not the public registry, so it needs to be updated to the correct image before release/merge